.husky/_/husky.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
Is this introduced by the new husky version?

Reply:
Looks like this is the recommended way to define husky config where before we had defined it in package.json.

Reply:
Yeah that's right, it's the result of `npx husky install`